### DISCUSSION TITLE
Can't enter anything in newtumbl account info

### DISCUSSION THREAD

**Discussion by jjdnl** [TumblThreeApp/TumblThree#350] [Q&A (Question & Answer)]
**Describe the bug**
I'm trying to enter the account info into for the newtumbl account, but can't type anything in the username or password fields

**To Reproduce**
1. Go to settings
2. Click on connection
3. Click on Authenticate for newtumbl account
4. In popup no information can be entered

**Expected behavior**
enter username and password

**Desktop (please complete the following information):**
 - TumblThree version: 2.8.0.347
 - OS: [e.g. Windows 10 Home] Windows 10 pro build 19044.1889
 - Browser [e.g. Chrome, Safari, IE, Edge] Firefox 
 - Version [e.g. 22] 104.0.2 (64-bit)
-
**Additional context**
Tumblr account can be authenticated

**Comment by thomas694** [comment]
Thanks for reporting a problem.
I had problems setting the focus into the fields when I clicked near the hint texts. It's a strange behavior, maybe related to javascript or the like. When I clicked in the big empty area on the right side, the textbox got the focus. Does this work for you too?

**Comment by jjdnl** [comment]
Yes that actually does work
